src/python/WMCore/WorkQueue/WorkQueueBackend.py

```diff
@@ -385,7 +385,7 @@ def availableWork(self, thresholds, siteJobCounts, teams = None, wfs = None):
                 elements.append(element)
                 if site not in siteJobCounts:
                     siteJobCounts[site] = {}
-                siteJobCounts[site][prio] = siteJobCounts[site].setdefault(prio, 0) + element['Jobs']
+                siteJobCounts[site][prio] = siteJobCounts[site].setdefault(prio, 0) + element['Jobs']*(1+element.get('BlowupFactor', 0.0))
```

Comment:
I'm reluctant to switch the defaults to 1.0; this way, there's less chance that coding errors would *shrink* the estimated number of jobs.